docs/remote_inference_blueprints/standard_blueprints/bedrock_connector_cohere_cohere.embed-english-v3_standard_blueprint.md

```diff
@@ -0,0 +1,190 @@
+    "credential": {
+        "access_key": "<PLEASE ADD YOUR AWS ACCESS KEY HERE>",
+        "secret_key": "<PLEASE ADD YOUR AWS SECRET KEY HERE>",
+        "session_token": "<PLEASE ADD YOUR AWS SECURITY TOKEN HERE>"
+    },
+    "parameters": {
+        "region": "<PLEASE ADD YOUR AWS REGION HERE>",
+        "service_name": "bedrock",
+        "truncate": "<NONE|START|END>",
+        "input_type": "<search_document|search_query|classification|clustering>",
+        "model": "cohere.embed-english-v3"
+    },
+    "actions": [
+        {
+            "action_type": "predict",
+            "method": "POST",
+            "headers": {
+                "x-amz-content-sha256": "required",
+                "content-type": "application/json"
+            },
+            "url": "https://bedrock-runtime.${parameters.region}.amazonaws.com/model/${parameters.model}/invoke",
+            "request_body": "{ \"texts\": ${parameters.texts}, \"truncate\": \"${parameters.truncate}\", \"input_type\": \"${parameters.input_type}\" }"
+        }
+    ]
+}
+```
+
+If using the AWS Opensearch Service, you can provide an IAM role arn that allows access to the bedrock service.
+Refer to this [AWS doc](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ml-amazon-connector.html)
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+    "name": "Amazon Bedrock Connector: Cohere embed-english-v3",
+    "description": "Test connector for Amazon Bedrock Cohere embed-english-v3 model",
+    "version": 1,
+    "protocol": "aws_sigv4",
+    "credential": {
+        "roleArn": "<PLEASE ADD YOUR AWS ROLE ARN HERE>"
+    },
+    "parameters": {
+      "region": "<PLEASE ADD YOUR AWS REGION HERE>",
+      "service_name": "bedrock",
+      "truncate": "<NONE|START|END>",
+      "input_type": "<search_document|search_query|classification|clustering>",
+      "model": "cohere.embed-english-v3"
+    },
+    "actions": [
+        {
+          "action_type": "predict",
+          "method": "POST",
+          "headers": {
+            "x-amz-content-sha256": "required",
+            "content-type": "application/json"
+          },
+          "url": "https://bedrock-runtime.${parameters.region}.amazonaws.com/model/${parameters.model}/invoke",
+          "request_body": "{ \"texts\": ${parameters.texts}, \"truncate\": \"${parameters.truncate}\", \"input_type\": \"${parameters.input_type}\" }"
+        }
+    ]
+}
+```
+
+Sample response:
+```json
+{
+  "connector_id": "hijwwZABNrAVdFa9prf7"
+}
+```
+For more information of the model inference parameters in the connector, please refer to this [AWS doc](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-embed.html)
+
+## 3. Create model group:
+
+```json
+POST /_plugins/_ml/model_groups/_register
+{
+    "name": "remote_model_group_cohere",
+    "description": "model group for cohere models"
+}
+```
+
+Sample response:
+```json
+{
+  "model_group_id": "IMobmY8B8aiZvtEZeO_i",
+  "status": "CREATED"
+}
+```
```

Comment:
Can this be removed, now that there is default model groups and these steps aren't required?

Reply:
model group is not required. but it's recommended for users to better manager their models, I will add the optional to `3. Create model group(Optional):`

Reply:
+1. Its good to add this step as optional to reduce the steps.